bounties/maven/reveal.js/1/README.md

```diff
@@ -0,0 +1,5 @@
+# Overview
+
+`org.webjars.bower:reveal.js` is a framework for easily creating presentations using HTML.
```

Comment:
The `blacklist` provided as fix after the issue was reported on https://hackerone.com/reports/691977 seems work and fix the issue as well, but I'm not sure of how safe it is (especially blacklisting approaches are sometimes bypassable).

In fact, the available methods for `Reveal` that could be called through the `postMessage API` are those ones: https://github.com/hakimel/reveal.js/blob/b6cc6b4916d594ac9f5aeed34d4c4c93dafc1a12/js/reveal.js#L5959-L6187 (in the `newest` all is imported by other files to make it smaller) ... are you able to provide a POC showing you're able to introduce XSS through one of the present methods?

Reply:
Removed this for now due to it being questionable, a new bounty will be put in its place. Cheers 😎 👍